intelmq/bots/collectors/xmpp/collector.py

```diff
@@ -61,7 +65,7 @@ def session_start(self, event):
                 self.logger.error('Server is taking too long to respond.')
                 self.disconnect()
 
-            if self.xmpp_room and self.plugin.get('xep_0045'):
+            if self.xmpp_room:  # and self.plugin.get('xep_0045') # this check should also exist!
```

Comment:
Somehow, the second check has stopped working and does not return any value. I don't know why. Although I didnt't test it it seems that this started after changing the behaviour how bots are loaded... (`BOT = xxx` instead of `__main__`). As I could not figure out why this happend, I left it in the code.